apps/api/src/domain/models/digital-text/entities/digital-text-page.entity.ts

```diff
@@ -32,17 +35,25 @@ export default class DigitalTextPage extends BaseDomainModel implements IDigital
      */
     readonly content?: MultilingualText;
 
+    @NestedDataType(MultilingualAudio, {
+        label: `multilingual audio`,
+        description: `maintains references to audio in available languages`,
+    })
+    readonly audio: MultilingualAudio;
```

Comment:
Don't forget to opt-in to multilingual audio for terms.

Reply:
[Here](https://www.pivotaltracker.com/story/show/186726351) is a story for that.